benchmarking/benchmark.py

```diff
@@ -0,0 +1,346 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_array
+
+import rdkit.Chem.rdFingerprintGenerator as fpgens
+from rdkit.Chem.rdMolDescriptors import GetMACCSKeysFingerprint
+from rdkit.Chem.rdReducedGraphs import GetErGFingerprint
+
+from time import time
+
+from featurizers.fingerprints import (
+    MorganFingerprint,
+    AtomPairFingerprint,
+    TopologicalTorsionFingerprint,
+    MACCSKeysFingerprint,
+    ERGFingerprint,
+)
+
+dataset_name = "ogbg-molhiv"
+N_SPLITS = 5
+N_REPEATS = 5
+N_CORES = [1, 2, 4, -1]
+COUNT_TYPES = [False, True]
+SPARSE_TYPES = [False, True]
+
+
+def get_times_emf(X, n_molecules, transformer_function, **kwargs):
+    result = []
+    emf_transformer = transformer_function(**kwargs)
+    for data_fraction in np.linspace(0, 1, N_SPLITS + 1)[1:]:
+        n = int(n_molecules * data_fraction)
+        subset = X[:n]
+        times = [None for _ in range(N_REPEATS)]
+        for i in range(N_REPEATS):
+            start = time()
+            X_transformed = emf_transformer.transform(subset)
+            end = time()
+            times[i] = end - start
+        result.append(sum(times) / N_REPEATS)
```

Comment:
You can use statistics.mean() or np.mean() instead of calculating mean yourself.

Reply:
Done